src/components/RightSidebar.tsx

```diff
@@ -0,0 +1,89 @@
+import { AnimatePresence, motion } from "framer-motion";
+import { ReactNode, useState } from "react";
+import { Css, Palette } from "src/Css";
+import { useTestIds } from "src/utils";
+import { IconKey } from "./Icon";
+import { IconButton } from "./IconButton";
+
+export type SidebarContentProps = {
+  icon: IconKey;
+  render: () => ReactNode;
+};
+
+export type RightSidebarProps = {
+  content: SidebarContentProps[];
+};
+
+export function RightSidebar({ content }: RightSidebarProps) {
+  const [selectedIcon, setSelectedIcon] = useState<IconKey | undefined>(undefined);
+  const tid = useTestIds({}, "rightSidebar");
+  const width = 380;
+
+  return (
+    <>
+      <AnimatePresence>
+        {selectedIcon === undefined && (
+          // Display the icons vertically
+          <motion.div
+            key="iconsWithoutSidebar"
+            animate={{ transition: { delay: 0.2, opacity: 1 } }}
```

Comment:
this delay doesn't seem to do anything. was trying to add a delay so the icons don't show up right away when you close the sidebar